addons/resource.language.en_gb/resources/strings.po

```diff
@@ -24340,7 +24340,19 @@ msgctxt "#39206"
 msgid "CC"
 msgstr ""
 
-#empty strings from id 39207 to 39999
+# Allow using Starfish decoder
+#: system/settings/linux.xml
+msgctxt "#39207"
+msgid "Allow using Starfish decoder"
+msgstr ""
+
+#. Description of setting with label #39207 "Allow hardware acceleration - Starfish"
+#: system/settings/linux.xml
+msgctxt "#39208"
+msgid "Enable Starfish decoding of video files"
```

Comment:
It won't be clear to many what "Starfish" is. The description could be enhanced to clarify this.

Reply:
It's pretty clear that a toggle button enables something, so maybe just devote the description so the user doesn't have to shell out to google.

"Starfish is the hardware video decoder for LG's webOS. Disable it for troubleshooting or testing."